jsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I guess it accepts both tsconfig.json and jsconfig?... Can we use the generated one from tsc --init? That way, we can quickly look up what's not enabled.